### PR TITLE
Add TLS version and cipher to the accessLog

### PR DESCRIPTION
**Pull request by na4ma4**
### What does this PR do?

Adds the TLS Version and Cipher to the logs when using JSON.

### Motivation

Knowing the TLS Version and Cipher of requests helps indicate when old ciphers and versions can be deprecated.

### More

- [x] Added/updated tests
- [x] Added/updated documentation

### Additional Notes

relates to issue #7421

### CODE REVIEW

**Comment by sylr** [comment]
I believe this is related to https://github.com/traefik/traefik/pull/6255 so maybe worth moving and making `getRequestTLSVersion` and `getRequestTLSCipher` public so that metrics and logs get coherent values.

File: pkg/middlewares/accesslog/logger_test.go [hunk 1]
```diff
@@ -355,6 +355,8 @@ func TestLoggerJSON(t *testing.T) {
 				Duration:                  assertFloat64NotZero(),
 				Overhead:                  assertFloat64NotZero(),
 				RetryAttempts:             assertFloat64(float64(testRetryAttempts)),
+				"TLSVersion":              assertString("TLS13"),
+				"TLSCipher":               assertString("TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256"),
```
Comment (jspdown) [inline pkg/middlewares/accesslog/logger_test.go:359]: Could you directly use `TLSVersion` and `TLSCipher` constants here?

**Comment by na4ma4** [comment]
Would it be better to move the `getRequestTLSVersion` and `getRequestTLSCipher` functions into [pkg/tls/tls.go](https://github.com/traefik/traefik/blob/master/pkg/tls/tls.go) as exported functions, rather than copy/pasting the code ?

Since it uses the data from there anyway.

**Review by kevinpollet — APPROVED** [review]
Thanks 👍

**Review by sylr — APPROVED** [review]

**Review by dtomcej — APPROVED** [review]
LGTM
:shipit:

**Review by jspdown — APPROVED** [review]
LGTM :+1:

**Pull request merged**